README.md

```diff
@@ -1,11 +1,10 @@
 # Minesweeper
 
-**TODO: Add description**
+A command-line Minesweeper game
```

Comment:
This README assumes the user has Elixir installed on their computer. I quick `brew install elixir` did the trick for me but still something to keep in mind when writing documentation. I was able to successfully run it following these direction though! 👍

Reply:
Thanks for letting me know, Kara!

The [docs](https://hexdocs.pm/mix/master/Mix.Tasks.Escript.Build.html) say "An escript can run on any machine that has Erlang/OTP installed and by default does not require Elixir to be installed, as Elixir is embedded as part of the escript." I'll add the Erlang/OTP requirement to the instructions! :)